query-engine/connector-test-kit/src/test/scala/writes/ids/IntIdUpdateSpec.scala

```diff
@@ -197,7 +197,7 @@ class IntIdUpdateSpec extends FlatSpec with Matchers with ApiSpecBase {
   }
 
 
-  "Updating a non-unique field of type Int with autoincrement" should "work"  taggedAs (IgnoreMySql)  in {
+  "Updating a non-unique field of type Int with autoincrement" should "work"  taggedAs (IgnoreSQLite, IgnoreMySql)  in {
```

Comment:
This should never have worked. The migration engine was generating completely wrong tables for these tests on SQLite.